provision.sh

```diff
@@ -37,25 +75,40 @@ if [ -n "$HTTPS_PROXY" ]; then
   echo "https_proxy=$HTTPS_PROXY" >> /etc/wgetrc
 fi
 
-echo "Installing Kong version: $KONG_VERSION"
 
-# Installing other dependencies
+echo "*************************************************************************"
+echo Setting up APT repositories
+echo "*************************************************************************"
+
+#postgres
+sudo add-apt-repository "deb https://apt.postgresql.org/pub/repos/apt/ precise-pgdg main"
+wget --quiet -O - https://postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+
+#cassandra
+echo 'deb http://debian.datastax.com/community stable main' | sudo tee -a /etc/apt/sources.list.d/cassandra.sources.list
+wget -q -O - '$@' http://debian.datastax.com/debian/repo_key | sudo apt-key add -
+
+
 sudo apt-get update
+
+echo "*************************************************************************"
+echo Installing APT packages
+echo "*************************************************************************"
+
+
+#generic usability utilities
+sudo apt-get install -y httpie jq
+
+#Installing required dependencies
 sudo apt-get install -y git curl make pkg-config unzip libpcre3-dev apt-transport-https language-pack-en libssl-dev
```

Comment:
Maybe we could drop libpcre3-dev here? Is libssl-dev necessary? Do Cassandra or Postgres or other external dependencies need it? Lua libraries installed for development purposes while working on Vagrant (not shipped with Kong itself) should link against the OpenSSL version shipped with Kong.

Reply:
Only wanted to change the setup, not the content. But tested and removed it anyway (some rudimentary manual tests)

Reply:
Seems like libssl-dev was one to keep - oops!